lib/hexo/findYarnRootWorkspace.ts

```diff
@@ -0,0 +1,57 @@
+import { existsSync, readFileSync } from 'hexo-fs';
+import micromatch from 'micromatch';
+import { dirname, join, normalize, relative } from 'path';
```

Comment:
Is there any existing library that can do this for us? This should be a common issue, so there has to be a library lying around that does this.

Reply:
pls wait

Reply:
> Is there any existing library that can do this for us? This should be a common issue, so there has to be a library lying around that does this.

done i remove it temporarily

Reply:
i move that improvisation to other branch https://github.com/dimaslanjaka/hexo/tree/v7-yarn-v3